src/helpers/filterInvalidEmails.ts

```diff
@@ -1,11 +1,9 @@
-import formatEmailDomain from './formatEmailDomain';
-
-function filterInvalidEmails(emailDomainInput: string, commitEmails: string[]): string[] {
-  const emailDomains = [];
-
-  const emailDomain = formatEmailDomain(emailDomainInput);
-  const invalidEmails = commitEmails.filter((email) => !email.endsWith(emailDomain));
-  emailDomains.push(emailDomain);
+function filterInvalidEmails(emailDomains: string[], commitEmails: string[]): string[] {
```

Comment:
You cannot just change the `emailDomainInput: string` into `emailDomains: string[]` (change a string to an array of strings) without changing the usage of the function in tests :slightly_smiling_face: